src/components/ShareStructure/EditShareStructure.vue

```diff
@@ -287,17 +294,17 @@ export default class EditShareStructure extends Mixins(CurrencyLookupMixin) {
   private getParValueRule (): Array<Function> {
     if (!this.hasNoParValue) {
       return [
-        v => !!v || 'Par value is required',
-        v => v > 0 || 'Amount must be greater than 0',
-        v => (v < 1) ? (/^(\d+(\.\d{0,3})?|\.\d{0,3})$/.test(v) || 'Amounts less than 1 can be entered with up to 3 decimal place')
+        (v: string) => !!v || 'Par value is required',
+        (v: string) => +v > 0 || 'Amount must be greater than 0',
```

Comment:
Note unary + operator to change string to a number (but since this was not previously a "strict" comparison, it still worked).